queue/abstract.lua

```diff
@@ -640,6 +664,13 @@ local function build_stats(space)
     return stats
 end
 
+--- identify the session.
```

Comment:
This description is a bit confusing for me. I would say something like 'Associate a current connection with given session'.

Reply:
Accepted.